common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -396,6 +396,10 @@ public class BrokerConfig extends BrokerIdentity {
 
     private boolean enableMixedMessageType = false;
 
+    private boolean enableSplitRegistration = false;
```

Comment:
Just a reminder to the users that this switch cannot be used together with deleteTopicWithBrokerRegistration, otherwise there will be a loss of routing.

Reply:
Fixed.